src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue

```diff
@@ -5,7 +5,7 @@
         :fade="true"
         :backdrop="true"
         :visible.sync="proxyVisible"
-        :disabled="!isAllValid"
+        :disabled="!isAllValid && !isLabelDuplicated && !externalItemsInvalidTexts.length"
```

Comment:
question: Perhaps... code below is correct?
```suggestion
        :disabled="!isAllValid || isLabelDuplicated || externalItemsInvalidTexts.length"
```

Reply:
Oh thanx!!